http-clients/src/main/java/com/palantir/remoting3/clients/ClientConfigurations.java

```diff
@@ -86,6 +86,28 @@ public static ClientConfiguration of(
                 .build();
     }
 
+    /**
+     * Creates a new {@link ClientConfiguration} instance from the given SSL configuration, URIs and proxy
+     * configuration, filling in all other configuration with the defaults specified as constants in this class.
+     */
+    public static ClientConfiguration of(
```

Comment:
can you not use the builder?

Reply:
Yeah I can also use the ClientConfiguration builder if the `createProxySelector` is public. I can either make that public, or add something like this